packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2054,7 +2054,11 @@ abstract class BuildContext {
 
   /// Determines if [widget] is in the process of updating.
   ///
-  /// This flag will return `true` if [widget] is in one of those life-cycles:
+  /// This flag will return `true` if [widget] is in a life-cycles that both:
+  /// - declaratively updates a tree ([Widget] tree, [RenderObject] tree)
+  /// - can consume an [InheritedWidget]
```

Comment:
What does this mean?  Subscribe?  We can subscribe to an InheritedWidget in many instances.  What restriction does this add to the other line that we're not already expressing?

Reply:
That's what exclude `State.initState`